app/battles/knights_preparing.py

```diff
@@ -0,0 +1,20 @@
+def preparing(temp_knight: dict) -> dict:
```

Comment:
Rename this file to `knights_prepraration.py` and the function to `prepare`, function name should represent action.